### PR TITLE
Add time diffs to logs of the db-analyser

### PR DESCRIPTION
**Pull request by kderme**
This helps keeping track of what happens when.

### CODE REVIEW

File: ouroboros-consensus-cardano/tools/db-analyser/Main.hs [hunk 1]
```diff
@@ -14,13 +14,13 @@ module Main (main) where
 import           Data.Foldable (asum)
 import           Options.Applicative
 
-import           Control.Tracer (contramap, debugTracer, nullTracer)
+import           Control.Tracer (Tracer(..), nullTracer)
```
Comment (mrBliss) [inline ouroboros-consensus-cardano/tools/db-analyser/Main.hs]: I think stylish-haskell will want a space before the opening paren.

**Review by mrBliss — APPROVED** [review]
Nice!

Can you show an example of this output? (Saves me the time to compile it)

**Comment by kderme** [comment]
> Nice!
> 
> Can you show an example of this output? (Saves me the time to compile it)

```
[6.881464507s] TraceLedgerReplayEvent (ReplayedBlock (RealPoint (SlotNo 388798) a4b1b68c312de308ec20bf8a3000fc60f51dd3496c8af1be6768cbeb70410775) (At (Block {blockPointSlot = SlotNo 388799, blockPointHash = 45c2185719fb3f94bd112e69bde2c006e593e6d8e06f6175fc351141034a83f7})))
[6.882013202s] TraceLedgerReplayEvent (ReplayedBlock (RealPoint (SlotNo 388799) 45c2185719fb3f94bd112e69bde2c006e593e6d8e06f6175fc351141034a83f7) (At (Block {blockPointSlot = SlotNo 388799, blockPointHash = 45c2185719fb3f94bd112e69bde2c006e593e6d8e06f6175fc351141034a83f7})))
[6.882338994s] TraceOpenEvent OpenedLgrDB
[6.882450386s] TraceOpenEvent (OpenedDB (At (Block {blockPointSlot = SlotNo 388799, blockPointHash = 45c2185719fb3f94bd112e69bde2c006e593e6d8e06f6175fc351141034a83f7})) (At (Block {blockPointSlot = SlotNo 388799, blockPointHash = 45c2185719fb3f94bd112e69bde2c006e593e6d8e06f6175fc351141034a83f7})))
ChainDB tip: At (Block {blockPointSlot = SlotNo 388799, blockPointHash = 45c2185719fb3f94bd112e69bde2c006e593e6d8e06f6175fc351141034a83f7})
[7.084296478s] TraceLedgerEvent (TookSnapshot (DiskSnapshot 24) (At (RealPoint (SlotNo 386634) 5d324d033103f0cf769c68c7a1e3dda0279317957e5f877c3fdfdd10c22b8ee3)))
[7.085151379s] TraceLedgerEvent (DeletedSnapshot (DiskSnapshot 22))
[7.085368065s] TraceImmDBEvent DBClosed
```

**Comment by mrBliss** [comment]
bors merge

**Pull request merged**